java-checks/src/main/java/org/sonar/java/checks/IndentationCheck.java

```diff
@@ -131,14 +131,17 @@ public void visitMethodInvocation(MethodInvocationTree tree) {
 
   @Override
   public void visitLambdaExpression(LambdaExpressionTree lambdaExpressionTree) {
-    int previousTokenLine = getPreviousToken(lambdaExpressionTree).line();
-    int lambdaFirstTokenLine = lambdaExpressionTree.firstToken().line();
-    if (previousTokenLine != lambdaFirstTokenLine) {
-      expectedLevel += indentationLevel;
-    }
-    super.visitLambdaExpression(lambdaExpressionTree);
-    if (previousTokenLine != lambdaFirstTokenLine) {
-      expectedLevel -= indentationLevel;
+    // scan lambdaExpressionTree.parameters() is useless
```

Comment:
Because parameter of lambda can not raise issue, it's just: type, identifier and no initializer. Indentation is checked only for member of "class", "block", "switch". A parameter of lambda can not contains one of those. Should I change the comment?